cli/ceph/orch/tuned_profile.py

```diff
@@ -0,0 +1,61 @@
+from cli import Cli
+from utility.log import Log
+
+log = Log(__name__)
+
+
+class TunedProfile(Cli):
+    def __init__(self, nodes, base_cmd):
+        super(TunedProfile, self).__init__(nodes)
+        self.base_cmd = f"{base_cmd} tuned-profile"
+
+    def apply(self, spec_file, **kw):
+        """
+        apply tuned profile.
+        Args:
+            spec_file: tune spec file details
+            kw (dict): Key/value pairs that needs to be provided to the installer.
```

Comment:
Remove this key if no optional parameters are supported.

Reply:
Updated